File: eBarberShop/eBarberShop.Model/Requests/RecenzijeInsertRequest.cs
namespace eBarberShop.Model.Requests
{
    public class RecenzijeInsertRequest
    {
        public string Sadrzaj { get; set; }
        public double Ocjena { get; set; }
        public DateTime DatumObjave { get; set; }
        public int KorisnikId { get; set; }
    }
}

File: eBarberShop/UI/ebarbershop_mobile/lib/models/recenzije/recenzije_insert_request.dart
import 'package:json_annotation/json_annotation.dart';

part 'recenzije_insert_request.g.dart';

@JsonSerializable()
class RecenzijeInsertRequest {
  String sadrzaj;
  double ocjena;
  DateTime datumObjave;
  int korisnikId;

  RecenzijeInsertRequest(
      this.sadrzaj, this.ocjena, this.datumObjave, this.korisnikId);

  factory RecenzijeInsertRequest.fromJson(Map<String, dynamic> json) =>
      _$RecenzijeInsertRequestFromJson(json);

  Map<String, dynamic> toJson() => _$RecenzijeInsertRequestToJson(this);
}